### ISSUE TITLE
Possible incompatibility with h5py 2.4.0

### ISSUE THREAD

**Issue by wasade** [biocore/biom-format#583]
Currently getting the following from `pick_open_reference_otus.py` on QIIME 1.9rc1 tutorial

```python
Traceback (most recent call last):
  File "/usr/local/bin/pick_open_reference_otus.py", line 450, in <module>
    main()
  File "/usr/local/bin/pick_open_reference_otus.py", line 429, in main
    minimum_failure_threshold=minimum_failure_threshold)
  File "/usr/local/lib/python2.7/dist-packages/qiime/workflow/pick_open_reference_otus.py", line 1076, in pick_subsampled_open_reference_otus
    table = Table.from_hdf5(biom_file)
  File "/usr/local/lib/python2.7/dist-packages/biom/table.py", line 3098, in from_hdf5
    id_ = h5grp.attrs['id']
AttributeError: 'file' object has no attribute 'attrs'
```

Cross ref with biocore/qiime#1847

**Comment by wasade** [comment]
Seeing this issue with h5py 2.3.1. Leaving this issue open as it is not clear whether this is upstream of BIOM or a fault in BIOM

**Comment by jairideout** [comment]
This was resolved in biocore/qiime#1853.
